src/debugging/netSdk/NetSdkDebugHelper.ts

```diff
@@ -0,0 +1,51 @@
+import { commands } from "vscode";
+import { NetChooseBuildTypeContext, netContainerBuild } from "../../scaffolding/wizard/net/NetContainerBuild";
+import { unresolveWorkspaceFolder } from "../../utils/resolveVariables";
+import { DockerDebugScaffoldContext } from "../DebugHelper";
+import { DockerDebugConfiguration } from "../DockerDebugConfigurationProvider";
+import { NetCoreDebugHelper, NetCoreDebugScaffoldingOptions } from "../netcore/NetCoreDebugHelper";
+
+const netContainerSdkBuildSymbol = 'dotnet-container-sdk: debug';
```

Comment:
Nit: File-level constants usually get PascalCase
```suggestion
const NetContainerSdkBuildSymbol = 'dotnet-container-sdk: debug';
```